build-tools/debs/kubeadm-10.conf

```diff
@@ -0,0 +1,8 @@
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--kubeconfig=/etc/kubernetes/kubelet.conf --require-kubeconfig=true"
+Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true"
+Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
+Environment="KUBELET_DNS_ARGS=--cluster-dns=10.96.0.10 --cluster-domain=cluster.local"
+Environment="KUBELET_EXTRA_ARGS=--v=4"
```

Comment:
I removed this in the debs, because that logging level is too noisy, and users want to be able to easily append `Environment="KUBELET_EXTRA_ARGS=--xxx=yyy"` to configure the kubelet